### DISCUSSION TITLE
Audio files from jspsych not playing on the browsers

### DISCUSSION THREAD

**Discussion by sasis001** [jspsych/jsPsych#1360] [Q&A]
Hi,

Firefox or other browsers are not preloading or playing audio files in any of the html files even after setting preferences to allow audio files. The messages I received are, "You have not given special permissions" and "this file is stored in the local computer." I was able to run all files with embedded audio files previously. 

Have you run into this issue and would you know a fix to this.

Thanks,
Jay

**Comment by jodeleeuw** [comment]
Hi Jay,

Can you clarify what has changed since you were able to run the audio files?

It sounds like you might be trying to play files before the user interacts with the page. Browsers don't allow audio to play unless the user interacts first (mouse click, key press, etc.). Do you have a trial that requires interaction before the audio?
